zerver/views/realm.py

```diff
@@ -341,6 +341,9 @@ def update_realm_user_settings_defaults(
         json_validator=check_int_in(UserProfile.COLOR_SCHEME_CHOICES), default=None
     ),
     translate_emoticons: Optional[bool] = REQ(json_validator=check_bool, default=None),
+    display_emoji_reaction_users: Optional[bool] = REQ(
+        json_validator=check_bool, default=None, documentation_pending=True
```

Comment:
`documentation_pending` is wrong -- this parameter is documented in `zulip.yaml`.